packages/boxel-ui/test-app/app/components/icons-grid.gts

```diff
@@ -0,0 +1,135 @@
+import Component from '@glimmer/component';
+import { ALL_ICON_COMPONENTS } from '@cardstack/boxel-icons/boxel-icons-meta';
+import { Tooltip } from '@cardstack/boxel-ui/components';
+import { tracked } from '@glimmer/tracking';
+import { on } from '@ember/modifier';
+
+function importStatement(name: string): string {
+  return `import ${toPascalCase(name)} from '@cardstack/boxel-icons/${name}';`;
+}
+
+function toPascalCase(text: string): string {
+  return text.replace(/(^\w|-\w)/g, clearAndUpper);
+}
+
+function clearAndUpper(text: string): string {
+  return text.replace(/-/, '').toUpperCase();
+}
+
+class Copyable {
+  @tracked recentlyCopied = false;
+  copyToClipboard = (text: string) => {
+    return (_ev: Event) => {
+      navigator.clipboard.writeText(text);
+      this.recentlyCopied = true;
+      setTimeout(() => {
+        this.recentlyCopied = false;
+      }, 2000);
+    };
+  };
+}
+
+function makeCopyable() {
+  return new Copyable();
+}
+
+export default class IconsGridComponent extends Component {
+  get allBoxelIconsComponents() {
+    return ALL_ICON_COMPONENTS.map((c) => {
+      return {
+        name: c.name,
+        component: c,
+      };
+    });
+  }
+  @tracked iconFilterString = '';
+  get boxelIconsComponents() {
+    return this.allBoxelIconsComponents.filter((c) => {
+      return c.name.toLowerCase().includes(this.iconFilterString.toLowerCase());
+    });
+  }
+  updateIconFilterString = (ev: Event) => {
+    this.iconFilterString = (ev.target as HTMLInputElement).value;
+  };
+
+  <template>
+    <div>
```

Comment:
Can this be removed? I think it’s causing the icons container to not take the whole height:

![Boxel Components 2024-10-11 09-15-28](https://github.com/user-attachments/assets/694d10b2-e4af-4069-89fd-04ffedec3cdc)

Reply:
I merged without changing this but am happy to revise. The challenge is that when you have "All" selected on the left, I don't think we want the icons to be listed in full. So I'm not sure how best to contain the grid while letting it breathe. Ideas welcome.